components/cf-layout/index.md

```diff
@@ -567,20 +567,56 @@ Main content...
 
 ## cf-grid helpers
 
-## Featured content module
-<div class="block">
-    <section class="block block__border block__flush featured-content-module">
-        <div class="featured-content-module_text">
-            <div class="category-slug">
-                <span class="featured-content-module_icon cf-icon cf-icon-speech-bubble">
-    </span> Featured </div>
-            <h2>Feature title</h2>
-            <p>Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur instructior ex pri. Cu pri inani constituto, cum aeque noster commodo cu.</p>
-            <a class="jump-link jump-link__underline"> <span class="jump-link_text">Read more about the feature</span> </a>
-        </div>
-        <div class="featured-content-module_visual"> <img class="featured-content-module_img" src="http://placekitten.com/g/540/270"> </div>
-    </section>
-</div>
+### Featured content module
+
+Featured content module, like a hero, consists of headline/text/optional call to action along with a visual. It is intended to be used in a main content column next to a sidebar.
+
+Text is full width & displayed above the visual in the default/mobile view.
+At larger screen sizes, the text occupies a fixed portion of the screen (equal to the width of 5 of 12 columns at 701px & 3 of 12 columns at 901px for desktop).
+The visual occupies the remaining space.
+The visual should be 640x360 (16x9 ratio) and resize to fit the height of the FCM with a static width and is anchored left when it becomes too wide for the available space.
+
+<section class="block block__border block__flush o-featured-content-module">
```

Comment:
Just realized this comment was never addressed => https://github.com/cfpb/capital-framework/pull/386#discussion_r75525312

Since this PR is just the sandbox and not the source, leave it as is for now, but we should add an issue to take a look at that block/element combo.